src/components/organisms/EarthEditor/CanvasArea/hooks.ts

```diff
@@ -99,8 +99,20 @@ export default (isBuilt?: boolean) => {
 
   // convert data
   const selectedLayerId = selected?.type === "layer" ? selected.layerId : undefined;
-  const layers = useMemo(() => convertLayers(layerData), [layerData]);
-  const selectedLayer = selectedLayerId ? layers?.findById(selectedLayerId) : undefined;
+
+  const rootLayer = useMemo(
+    () =>
+      !(
+        !layerData ||
+        !layerData.node ||
+        layerData.node.__typename !== "Scene" ||
+        !layerData.node.rootLayer
+      )
```

Comment:
Maybe this is just personal preference, but I feel like its harder to understand whats happening doing `!(!something || !somethingelse .. )` than doing `!!(something || somethingelse...)`. That way, there is less negating and converting we have to do in our heads.
If you agree, could this be re-written like: 
```suggestion
      !!(
        layerData?.node ||
        layerData.node.__typename === "Scene" ||
        layerData.node.rootLayer
      )
```
I removed the first `layerData ||` and just add the question mark to the first check. Please make sure that is actually okay.